network-api/networkapi/wagtailpages/templates/wagtailpages/bannered_campaign_page.html

```diff
@@ -23,3 +23,45 @@
     </div>
   </div>
 {% endblock %}
+
+The following block has been borrowed from the templates/tags/cta/petition.html file,
+rather than using the `cta` tag from our mini_site_tags collection, because the cta
+tag was written in the assumption that models would only ever have a single CTA field
+named "cta", which is an assumption that the bannered campaign page squarely violates.
+
+As such, the CTA tag needs to be refactored to take this into account, however this
+work should not be done at the same time as adding signup CTAs to the b.c. pages.
+
+
+{% block cta %}
+  {% if page.cta != None or page.signup != None %}
+  <div class="cta
+    {% if page.specific.narrowed_page_content == False %}
```

Comment:
Possibly out of scope, but I think this can be removed/made just one class- since if you're seeing the right column, the page content width is a fixed size anyway.

Reply:
I'd say out of scope - it's a copy of what we already had so I'd rather leave it the same as much as possible until we tackle the cleanup.

Reply:
Does the templating support wrapping `block` tags inside the `if` statement? Or can you call a block's super content manually? Just trying to reduce the WETness here.

Reply:
any toplevel block declaration means "I am overriding the content of this block: ..." and any block declaration inside of templating code means "I am defining a block here, that extending templates can override".

Reply:
Right, I wasn't sure if there was a secret manual call to `super`. Not a big deal, when you land this file a ticket to remember to DRY it out.